### PR TITLE
Support PHP 8.0

### PR DESCRIPTION
**Pull request by bepsvpt**
This pull request renames `Mixed` class to `MixedScalar` to support PHP 8.0.

GitHub Actions Workflow also has updated.

It will be a breaking change.

### CODE REVIEW

File: .github/workflows/continuous-integration.yml [hunk 1]
```diff
@@ -51,31 +51,25 @@ jobs:
...
 
     runs-on: ubuntu-latest
 
     strategy:
       matrix:
-        php-version:
-          - 7.2
-          - 7.3
-          - 7.4
+        php-version: ['7.2', '7.3', '7.4', '8.0']
```
Comment (spawnia) [inline .github/workflows/continuous-integration.yml]: I prefer the longhand list style

File: .github/workflows/continuous-integration.yml [hunk 2]
```diff
@@ -84,17 +78,8 @@ jobs:
...
-        run: composer update --prefer-lowest --no-interaction --no-progress --no-suggest
-
-      - name: "Install locked dependencies with composer"
-        if: matrix.dependencies == 'locked'
-        run: composer install --no-interaction --no-progress --no-suggest
-
-      - name: "Install highest dependencies with composer"
-        if: matrix.dependencies == 'highest'
-        run: composer update --no-interaction --no-progress --no-suggest
+      - name: "Install dependencies with composer"
```
Comment (spawnia) [inline .github/workflows/continuous-integration.yml:87]: Why did you change this part?

File: README.md [hunk 3]
```diff
@@ -31,7 +31,7 @@ A [RFC 5321](https://tools.ietf.org/html/rfc5321) compliant email.
 
 Arbitrary data encoded in JavaScript Object Notation. See https://www.json.org/.
 
-### [Mixed](src/Mixed.php)
+### [MixedScalar](src/MixedScalar.php)
```
Comment (spawnia) [inline README.md]: The GraphQL name of the type can still be `Mixed`, only the PHP class has to change.

**Review by spawnia — CHANGES_REQUESTED** [review]
Thanks for tackling this

File: .github/workflows/continuous-integration.yml (hunk 2 above)
Comment (bepsvpt) [reply .github/workflows/continuous-integration.yml:87]: `matrix.dependencies == 'locked'` will not work if `composer.lock` is not committed.

Thus, only `lowest` and `highest` will actually work. Besides, all dependencies are use stable version, `highest` is equal to `--prefer-stable`.

Thus, I simplified this section.

**Comment by spawnia** [comment]
@bepsvpt please allow me to edit the PR.

**Comment by bepsvpt** [comment]
Hi @spawnia,

It seems there is no such option on this page. I'm following https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork page steps but there is no such checkbox.

**Comment by spawnia** [comment]
Thank you
